features/order/managing_orders/handling_order_shipment/resending_shipment_confirmation_email.feature

```diff
@@ -17,17 +17,27 @@ Feature: Resending a shipment confirmation email for a chosen order
         And this order has already been shipped
         And I am logged in as an administrator
 
-    @ui @email
+    @ui @email @api
     Scenario: Resending a shipment confirmation email for a given order
         When I view the summary of the order "#00000666"
         And I resend the shipment confirmation email
         Then I should be notified that the shipment confirmation email has been successfully resent to the customer
         And an email with the shipment's confirmation of the order "#00000666" should be sent to "lucy@teamlucifer.com"
 
-    @ui @email
+    @ui @email @api
     Scenario: Resending a shipment confirmation email after shipping an order in different locale than the default one
         Given the order "#00000666" has been placed in "Polish (Poland)" locale
         When I view the summary of the order "#00000666"
         And I resend the shipment confirmation email
         Then I should be notified that the shipment confirmation email has been successfully resent to the customer
         And an email with the shipment's confirmation of the order "#00000666" should be sent to "lucy@teamlucifer.com" in "Polish (Poland)" locale
+
+    @ui @email @api
+    Scenario: Not being able to resend a confirmation email for a given shipment with wrong state
+        Given this customer placed another order "#00000023"
+        And the customer bought a single "Angel T-Shirt"
+        And the customer "Lucifer Morningstar" addressed it to "Seaside Fwy", "90802" "Los Angeles" in the "United States" with identical billing address
+        And the customer chose "Free" shipping method with "Cash on Delivery" payment
+        When I view the summary of the order "#00000023"
+        Then I should not be able to resend the shipment confirmation email
+        And an email with the shipment's confirmation of the order "#00000023" should not be sent to "lucy@teamlucifer.com"
```

Comment:
I'm not sure if this step is necessary, the one above should be enough as in UI we are unable to click the button, but in API we are checking the response